HotPathAllocationAnalyzer.Configuration/ConfigurationReader.cs

```diff
@@ -35,6 +35,15 @@ public ConfigurationReader(string configurationProjectDirectory)
             _configurationProject = analyzer.AddToWorkspace(workspace);
         }
 
+        public static IEnumerable<string> GenerateDisclaimer()
+        {
+            var disclaimer = "#  This file is automatically generated by the AllocationConfiguration class, DO NOT MODIFY IT MANUALLY it is overrided every time the configuration project is compiled";
```

Comment:
All classes that implement  `AllocationConfiguration` are used to generate the whitelist file